lib-python/littleboxes/xword.py

```diff
@@ -1,22 +1,24 @@
 import logging
 import puz
 
+
```

Comment:
What is the formatting styler you are using? I should probably set mine up to standardize formatting before committing.